src/main/java/nl/tudelft/contextproject/controller/GameState.java

```diff
@@ -4,29 +4,16 @@
  * Enum representing the state of the game.
  */
 public enum GameState {
-	/**
-	 * Waiting for the game to start.
-	 */
 	WAITING,
-	
-	/**
-	 * The main game is running.
-	 */
 	RUNNING,
-	
-	/**
-	 * The main game is suspended.
-	 */
 	PAUSED,
-	
-	/**
-	 * The main game is finished.
-	 */
 	ENDED;
```

Comment:
I think the above comments were somewhat useful, and don't neccesarily had to be removed.